src/mono/mono/mini/interp/transform.c

```diff
@@ -8441,7 +8511,7 @@ interp_reorder_bblocks (TransformData *td)
 				}
 			}
 		} else if (first->opcode == MINT_BR) {
-			// All bblocks jumping into this bblock can jump directly into the br target
+			// All bblocks jumping into this bblock can jump directly into the br target only if it is a single instruction in a bb
```

Comment:
The comment addition seems confusing. Did you mean to say `since it is the single instruction of the bb`

Reply:
Updated. It was confusing and left over from previous changes.